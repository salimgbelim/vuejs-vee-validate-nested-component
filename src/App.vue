<template>
  <div id="app" class="container">
      <div class="columns is-multiline">
        <simple-component></simple-component>
      </div>
  </div>
</template>

<script>
  import SimpleComponent from './components/SimpleComponent.vue';

  export default {
    name: 'app',
    components: {
      SimpleComponent
    }
  }
</script>

<style>
  #app {
    font-family: 'Avenir', Helvetica, Arial, sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    text-align: center;
    color: #2c3e50;
    margin-top: 60px;
  }
</style>
