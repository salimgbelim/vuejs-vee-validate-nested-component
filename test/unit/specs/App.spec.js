import Vue from 'vue';
import App from '@/App';

describe('App.test.js', () => {

  let cmp, vm;

  beforeEach(() => {
    cmp = Vue.extend(App);

    vm = new cmp({
      data: {
        messages: ['Cat']
      }
    }).$mount();

  });

  it('equals message to ["Cat"]', () => {
    expect(vm.messages).toEqual(['Cat']);
  });

  it('has the expected html structure', () => {
    expect(vm.$el).toMatchSnapshot();
  });
});
