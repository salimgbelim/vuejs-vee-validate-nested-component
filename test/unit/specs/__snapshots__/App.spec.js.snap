// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.test.js has the expected html structure 1`] = `
<div
  class="container"
  id="app"
>
  <ul>
    <li
      class="message"
      style="margin-top: 10px;"
    >
      
  Cat -- Paco

    </li>
  </ul>
</div>
`;
